src/views/fun-math/fun-math.vue.ts

```diff
@@ -0,0 +1,2 @@
+import Vue from 'vue';
+export default class FunMath extends Vue {}
```

Comment:
When you start working on this view, make sure you don't forget having `@Component`